### PR TITLE
US005-View Events

### PR DESCRIPTION
**Pull request by andrewmil**
An `"events"` field is now returned from the `collection-exercises/<short_name>/<period>` endpoint.
This is retrieved from the new (not yet implemented) `/events` endpoint in the collection exercise service and formatted in a useful way for the UI

The `/events` endpoint will be implemented as part of [US004](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/168132632/US004+Load+Events+L)

[Trello](https://trello.com/c/ZtQUGyXs/34-us005-view-events-25pts-l-5-days)
[Confluence](https://digitaleq.atlassian.net/wiki/spaces/RASB/pages/160235597/US005+View+Events+L)

### CODE REVIEW

**Comment by andrewmil** [comment]
[US005: View Events (25pts = L = 5 days)](https://trello.com/c/ZtQUGyXs/34-us005-view-events-25pts-l-5-days)

File: ras_backstage/resources/collection_exercise/single_collection_exercise.py [hunk 1]
```diff
@@ -24,15 +25,24 @@ def get(short_name, period):
...
         exercise = get_collection_exercise_by_period(exercises, period)
         if not exercise:
             return make_response(jsonify({"message": "Collection exercise not found"}), 404)
         full_exercise = collection_exercise_controller.get_collection_exercise_by_id(exercise['id'])
 
+        # Retrieve the events and format correctly
+        exercise_events = collection_exercise_controller.get_collection_exercise_events(exercise['id'])
+        formatted_events = {}
+        for event in exercise_events:
+            formatted_event_datetime = convert_event_to_new_format(event)
```
Comment (benjefferies) [inline ras_backstage/resources/collection_exercise/single_collection_exercise.py]: Feels like we should be returning an ISO 8601 date here and leaving it to the client to decide the format the date should be shown in as it's more of a presentation thing

Comment (andrewmil) [reply ras_backstage/resources/collection_exercise/single_collection_exercise.py]: Yeah that does feel better.
I've generally been trying to do the manipulation on the data that the UI is using in backstage but in this case it's probably better in the UI

Comment (andrewmil) [reply ras_backstage/resources/collection_exercise/single_collection_exercise.py]: Done

**Pull request merged**